lib/stat_tracker.rb

```diff
@@ -27,13 +27,32 @@ def create_teams(teams)
 			franchise_id = row[:franchiseid]
 			team_name = row[:teamname]
 			abbreviation = row[:abbreviation]
-			stadium = row[:stadium] # do symbols always return all lowercase or the same case as we assign it???
+			stadium = row[:stadium] # do symbols always return all lowercase or the same cse as we assign it???
```

Comment:
symbols will remain whatever case we write them in when we create the symbol in our code.